packages/ios-client/README.md

```diff
@@ -0,0 +1,59 @@
+# Fishjam iOS Client
+
+[Fishjam](https://github.com/fishjam-dev/fishjam) Client library for iOS apps written in Swift.
```

Comment:
nit: link can be updated to Fishjam Cloud (but we can also cleanup readme's in other PR)

Reply:
i suggest another PR